packages/core/types/client.d.ts

```diff
@@ -29,6 +29,11 @@ declare class Client {
     onError?: common.OnError,
     cb?: (err: any, event: Event) => void,
   ): void;
+  public _notify(
```

Comment:
This isn't intended to be part of the public interface, but is required for plugins which want to report unhandled errors.